package.json

```diff
@@ -5,29 +5,49 @@
   "main": "index.js",
   "scripts": {
     "create": "webpack",
-    "start": "webpack-dev-server --open"
+    "start": "webpack-dev-server --open",
+    "test": "eslint .",
+    "lintFix": "eslint --fix ."
   },
   "author": "Ori Levitan",
   "babel": {
     "presets": [
       "env",
       "react"
+    ],
+    "plugins": [
+      "transform-es2015-destructuring",
+      "transform-object-rest-spread",
+      "transform-decorators-legacy"
     ]
   },
   "license": "ISC",
   "devDependencies": {
     "babel-core": "^6.26.0",
+    "babel-eslint": "^8.2.1",
     "babel-loader": "^7.1.2",
+    "babel-plugin-transform-decorators-legacy": "^1.3.4",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "babel-preset-env": "^1.6.1",
     "babel-preset-react": "^6.24.1",
     "css-loader": "^0.28.8",
+    "eslint": "^4.15.0",
+    "eslint-config-airbnb": "^16.1.0",
+    "eslint-config-standard": "^11.0.0-beta.0",
```

Comment:
Pretty sure you don't need both 'standard' and 'airbnb' configs.

Reply:
I'm not sure how did they end up there together.

done